src/domain/orders/edit/modal.tsx

```diff
@@ -367,43 +365,41 @@ type OrderEditModalContainerProps = {
   order: Order
 }
 
+let isRequestRunningFlag = false
```

Comment:
Suggestion : can’t we use the isLoading from the hook instead?

Reply:
It won't be the same behaviour, since `isLoading` is set asynchronously, the other request will be fired.

Reply:
Ok got it. Thanks for the info 💪